lib/bin/lwm2m_carrier/os/lwm2m_os.c

```diff
@@ -90,7 +88,7 @@ void lwm2m_os_sys_reset(void)
 
 uint32_t lwm2m_os_rand_get(void)
```

Comment:
Should perhaps change name of this function to cycle32 so that we do not later expect this to be rand when coding in the lib?

Reply:
No, we should avoid changing the glue functions' names, for compatibility.

Reply:
Shouldn't the rand function be okay to use again after #2519 was merged?